test/functional/utils-specs.js

```diff
@@ -0,0 +1,87 @@
+import { fs, mkdirp } from 'appium-support';
+import nodefs from 'fs';
+import { copyGradleProjectRecursively } from '../../lib/utils';
+import os from 'os';
+import path from 'path';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+
+chai.should();
+chai.use(chaiAsPromised);
+const expect = chai.expect;
+
+describe('copyGradleProjectRecursively', function () {
+  let baseSrcDir;
+  let baseDestDir;
+
+  function expectNotExist (dir) {
+    expect(function () {
+      fs.accessSync(dir, fs.constants.F_OK);
+    }).to.throw();
```

Comment:
I wasn't able to write this check using `async/await`+`chai-as-promised`, so I had to you sync version of the function. It is ok for me, if someone can rewrite it to be async.

Reply:
use `promiseFunc.should.eventually.be.rejected/rejectedWith(/pattern/)`